infrastructure/persistence/LocalGameRepository.ts

```diff
@@ -1,8 +1,77 @@
 import { GameRepository } from "../../domain/repository";
-import { EntityMutation, Game } from "../../domain/entity";
+import {
+    ActionSpace,
+    Dwarf,
+    DwarfId,
+    EntityMutation,
+    Furnishing,
+    Game,
+    isMutationOfType,
+    Mutation,
+    Player,
+} from "../../domain/entity";
+import {
+    StoredActionSpace,
+    StoredDwarf,
+    StoredFurnishing,
+    StoredGame,
+    StoredPlayer,
+} from "./entity/entities";
 
+// Dummy in-memory repository for POC
 export class LocalGameRepository implements GameRepository {
-    applyMutation = (mutation: EntityMutation): void => {};
+    game: StoredGame = new StoredGame();
+    actionSpaces: Map<string, StoredActionSpace> = new Map();
+    furnishings: Map<string, StoredFurnishing> = new Map();
+    players: Map<string, StoredPlayer> = new Map();
+    dwarfs: Map<string, StoredDwarf> = new Map();
+
+    applyMutation = (mutation: EntityMutation): void => {
```

Comment:
[Detail] It would maybe be nice if it crashed during the compilation if we forget to implement it here when we add a new mutation type :thinking: